File: app/src/main/java/com/example/sarahn/waterreminderapp/classes/ClsOnValueChangeListmer.java
package com.example.sarahn.waterreminderapp.classes;

import android.content.Context;
import android.widget.NumberPicker;

/**
 * Created by SarahN on 6/13/2017.
 */
public class ClsOnValueChangeListmer implements NumberPicker.OnValueChangeListener {


    @Override
    public void onValueChange(NumberPicker picker, int oldVal, int newVal) {
        if(ClsBottomSheetActivity.values.length == 0){
            return;
        }else{
            ClsBottomSheetActivity.tv.setText(ClsBottomSheetActivity.values[newVal-1]);
        }
    }


}

File: app/src/main/java/com/example/sarahn/waterreminderapp/classes/ClsOnValueChangeListmer.kt
package com.example.sarahn.waterreminderapp.classes

import android.widget.NumberPicker
import android.widget.NumberPicker.OnValueChangeListener

/**
 * Created by SarahN on 6/13/2017.
 */
class ClsOnValueChangeListmer : OnValueChangeListener {
    override fun onValueChange(picker: NumberPicker, oldVal: Int, newVal: Int) {
        if (ClsBottomSheetActivity.values?.size == 0) {
            return
        } else {
            ClsBottomSheetActivity.tv!!.text = ClsBottomSheetActivity.values?.get(newVal - 1) ?: ""
        }
    }
}